src/transactions/ParallelApplyUtils.h

```diff
@@ -102,11 +266,17 @@ class LedgerAccessHelper
     getLedgerEntryOpt(LedgerKey const& key) = 0;
 
     // upsert returns true if the entry was created, false if it was updated.
+    // "created" here is interpreted narrowly to mean there was no live entry in
```

Comment:
nit: I'm not sure it's correct to use the term 'live' entry as it may imply that we can insert an entry if it has been evicted, which shouldn't be true. Typically we use 'live' vs 'archived' to distinguish entries w.r.t. their TTL state.

Reply:
oh, yeah, I will pick a different term. perhaps "populated".